src/cascade/saver/save_model_results.py

```diff
@@ -0,0 +1,92 @@
+"""
+Saves results from a model fit to a database which the EpiViz can access.
+"""
+import logging
+
+import save_results._save_results as sr
+
+from cascade.saver.generate_draws import generate_draws_table
+
+from cascade.saver import DRAWS_INPUT_FILE_PATTERN, MODEL_TITLE
+
+
+CODELOG = logging.getLogger(__name__)
+MATHLOG = logging.getLogger(__name__)
+
+
+def save_model_results(execution_context):
+    """
+    We want to visualize model results using EpiViz.  To do this, we need to
+    generate draws from a model fit, and save those draws in a database
+    that EpiViz can access.
+
+    Args:
+        execution_context (ExecutionContext): contains model id data
+
+    Returns:
+        (str) of the mvid returned by save_results
+
+    """
+
+    dm_file = execution_context.dismodfile
+
+    if not dm_file:
+        raise ValueError("Must provide a dismod file")
+
+    draws_df = generate_draws_table(dm_file)
```

Comment:
I'd like to keep the pattern where we drive database access code from outside. So you would have a method that takes a context, calls `generate_draws_table` and then passes the resulting table into this function. Does that make sense? Code that has direct database access stuff in it is always structured as utility functions and the sequences that drive the programs behavior are in higher level functions.